### PR TITLE
Finish task 5

### PR DESCRIPTION
**Pull request by karlrivkpmg**

### CODE REVIEW

**Comment by karlrivkpmg** [comment]
Ended up with a tiny conflict, does this need resolved first?

**Comment by karlrivkpmg** [comment]
Think I need the PR approved before I can fix the conflict?

File: server/__tests__/app.test.js [hunk 1]
```diff
@@ -74,3 +74,43 @@ afterAll(() => {
...
+
+  describe('4. GET /api/reviews/:review_id', () =>{
+
+    test("status:200, returns a specific review object matching the parametric review_id", ()=>{
+        return request(app)
+        .get('/api/reviews/1')
+        .expect(200)
+        .then((response)=>{
+            const review = response.body.review;
+            console.log(review);
```
Comment (Dutton1990) [inline server/__tests__/app.test.js:86]: dont forget to remove console logs!

**Review by Dutton1990 — APPROVED** [review]
Really nice Karl, I can't spot anything wrong here. Nice clean code and errors well handled, great job feel free to merge!

**Pull request merged**